exonum-java-binding-common/src/main/java/com/exonum/binding/common/proofs/list/CheckedListProof.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.common.proofs.list;
+
+import com.exonum.binding.common.hash.HashCode;
+import com.exonum.binding.common.proofs.common.ProofStatus;
+import java.util.NavigableMap;
+
+/**
+ * A checked list proof.
+ * In case of incorrect proof all methods (except for getStatus and compareWithRootHash)
```

Comment:
Why do we want to be able to call `compareWithRootHash` in case of incorrect proof?

Reply:
:+1: ,

Also, there isn't such a method in this interface — and I don't think we shall add one. @MakarovS , @DENMROOT , shan't we remove it from `CheckedMapProof`?

Reply:
there is no compareWithRootHash() method in CheckedListProof - incorrect javadoc @MakarovS